packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -40,30 +40,35 @@ export interface ChainAdapterArgs {
     ws: bitcoin.ws.Client
   }
   coinName: string
+  coinType: ChainTypes
+  curve?: string
+  bip44Params: BIP44Params
+
```

Comment:
- type of chain !== type of coin; granted this probably doesn't matter for UTXOs but it's an important semantic distinction
- curve shouldn't be a parameter; theoretically it should be looked up from `chainType`, but everything is `secp256k1` and we're going to have bigger fish to fry if we ever need to support another, so I'd leave it hardcoded on line 96(old)/102(new)
- note that the bip44Params are *only* defaults, should be named as such
```suggestion
  chainType: ChainTypes
  defaultBIP44Params: BIP44Params
```